src/componensts/Person/Person.jsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './Person.scss';
+
+export const Person = ({ perInfo }) => {
+  const {
+    name,
+    age,
+    sex,
+    isMarried,
+    partnerName,
+  } = perInfo;
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">
+        {`My name is ${name}`}
+      </h2>
+
+      {age && (
+        <p className="Person__age">
+          {`i am ${age}`}
+        </p>
+      )}
+
+      <p className="Person__partner">
+
+        {isMarried
+          ? (`My ${sex === 'f' ? 'husband' : 'wife'}'s is ${partnerName}`)
```

Comment:
Will be better to create separate variable for storing partnerSex and use this variable here in jsx